src/util/nodelist-to-array.js

```diff
@@ -0,0 +1 @@
+export default (nodeList) => Array.prototype.slice(nodeList);
```

Comment:
`Array.prototype.slice.call(nodeList)`?

Also nit: parenthesis aren't needed in single argument arrow functions.